packages/web-app-files/src/components/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -1,17 +1,28 @@
 <template>
   <oc-grid gutter="small" child-width="1-1">
-    <roles-select
-      mode="collaborators"
-      :roles="roles"
-      :selected-role="role"
-      @roleSelected="selectRole"
-    />
+    <label tag="label" class="oc-label">
```

Comment:
Is `<label tag="label"` valid or duplicate information? (Like <progress role="progress" or <nav role="navigation")

Reply:
Ah, I had originally `<translate tag="label"...` and forgot to remove that after switching back to the label. Will remove it

Reply:
Happens someplace else in this PR btw :)